tests/pkg_libcose/tests/01-run.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import sys
+from testrunner import run
+
+
+def testfunc(child):
+    child.expect_exact('OK (3 tests)')
```

Comment:
Maybe make this a regex. This way the test is extensible (and might not fail randomly ;-))

```suggestion
    child.expecr(r'OK (\d+ tests)')
```

Reply:
Agreed, Fixed